tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "noEmit": true,
     "resolveJsonModule": true,
     "strict": true,
-    "noUnusedLocals": true
+    "noUnusedLocals": true,
+    "types": []
```

Comment:
TypeScript was loading `@types/node` for some reason and that made `setTimeout` go crazy. We don't have any dependencies with `@types/...` so its empty for now.